src/styles/shared/_interaction-state.scss

```diff
@@ -24,3 +24,16 @@
   }
   background-image: $backgrounds;
 }
+
+@mixin list-single-selection-state {
+  background: var(--p-interactive);
+  content: '';
```

Comment:
Very nitpicky but if we are dealing with pseudo elements, I prefer to see `content` come first, since it is so vitally important to the pseudo element rendering.